themes/bootstrap3/templates/externalauth/ezproxylogin.phtml

```diff
@@ -0,0 +1,18 @@
+<?
+    // Set page title
+    $this->headTitle($this->translate('external_auth_heading'));
+?>
+
+<div class="row external-content-access">
+  <div class="col-sm-12">
+    <?=$this->flashmessages()?>
+    <? if ($this->unauthorized): ?>
+      <div class="unauthorized-description">
+        <p><?=$this->transEsc('external_auth_unauthorized_desc'); ?></p>
+      </div>
+      <div>
+        <a href="<?=$this->url('myresearch-logout')?>" class="logout<?=$logoutClass?> btn btn-primary" title="<?=$this->transEsc("Log Out")?>"><strong><?=$this->transEsc("Log Out")?></strong></a>
```

Comment:
What is the purpose of the $logoutClass variable here? Is that something specific to your local instance?